lib/temporal/activity/poller.rb

```diff
@@ -90,6 +91,8 @@ def poll_for_task
 
         Temporal::ErrorHandler.handle(error, config)
 
+        sleep_before_retry(poll_retry_seconds) unless poll_retry_seconds.zero?
```

Comment:
nit: You don't really need the `poll_retry_seconds.zero?` check, as `sleep(0)` is a valid command and won't have any effect